projects/packages/forms/src/dashboard/class-dashboard.php

```diff
@@ -247,4 +155,57 @@ public function has_feedback() {
 
 		return $posts->found_posts > 0;
 	}
+
+	/**
+	 * Returns url of forms admin page.
+	 *
+	 * @param string|null $tab Tab to open in the forms admin page.
+	 *
+	 * @return string
+	 */
+	public static function get_forms_admin_url( $tab = null ) {
+		$base_url = get_admin_url() . 'admin.php?page=jetpack-forms-admin';
+
+		return self::append_tab_to_url( $base_url, $tab );
+	}
+
+	/**
+	 * Appends the appropriate tab parameter to the URL based on the view type.
+	 *
+	 * @param string $url              Base URL to append to.
+	 * @param string $tab              Tab to open.
+	 *
+	 * @return string
+	 */
+	private static function append_tab_to_url( $url, $tab ) {
+		if ( ! $tab ) {
+			return $url;
+		}
+
+		$status_map = array(
```

Comment:
Why do we need this mapping here when we're using the exact same string we receive in the $tab param? Shouldn't something like this do the same job?
```php
    $valid_tabs = array( 'spam', 'inbox', 'trash' ):
    if ( ! in_array( $tab, $valid_tabs ) ) {
        return $url;
    }

    return $url . '#/responses?status=' . $status_map[ $tab ];
}